### PR TITLE
Forward port 7.15.0 changelog to 7.x

### PR DESCRIPTION
**Pull request by andresrc**
See #28033. Part of 7.15.0 release.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,100 @@
...
+*Affecting all Beats*
+
+- Improve `perfmon` metricset performance. {pull}26886[26886]
+- Preserve annotations in a kubernetes namespace metadata {pull}27045[27045]
+- Fix build constraint that caused issues with doc builds. {pull}27381[27381]
+- Do not try to load ILM policy if `check_exists` is `false`. {pull}27508[27508] {issue}26322[26322]
+- Fix bug with cgroups hierarchy override path in cgroups {pull}27620[27620]
+- Beat `setup kibana` command may use the elasticsearch API key defined in `output.elasticsearch.api_key`. {issue}24015[24015] {pull}27540[27540]
+- Fix `decode_xml` handling of array merging when using `to_lower: true`. {pull}27922[27922]
+- Seperate namespaces for V1 and V2 controller paths {pull}27676[27676]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:41]:
```suggestion
- Separate namespaces for V1 and V2 controller paths {pull}27676[27676]
```

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -3,6 +3,100 @@
...
+- Fix build constraint that caused issues with doc builds. {pull}27381[27381]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:42]: Should this be included?

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -3,6 +3,100 @@
...
+- Do not try to load ILM policy if `check_exists` is `false`. {pull}27508[27508] {issue}26322[26322]
+- Kubernetes autodiscover fails in node scope if node name cannot be discovered {pull}26947[26947]
+
+*Auditbeat*
+
+- file_integrity: honor include_files when doing initial scan. {issue}27273[27273] {pull}27722[27722]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:48]:
```suggestion
- File Integrity module: Honor `include_files` when doing an initial scan. {issue}27273[27273] {pull}27722[27722]
```

**Review by ruflin — APPROVED** [review]

File: CHANGELOG.asciidoc [hunk 4]
```diff
@@ -3,6 +3,100 @@
...
+*Winlogbeat*
+
+- Fix an issue with message template caching in the `wineventlog-experimental` API implementation. {pull}26826[26826]
+
+==== Added
+
+*Affecting all Beats*
+
+- Add proxy support for AWS functions. {pull}26832[26832]
+- Added policies to the elasticsearch output for non indexible events {pull}26952[26952]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:69]:
```suggestion
- Added policies to the Elasticsearch output for non indexible events {pull}26952[26952]
```

File: CHANGELOG.asciidoc [hunk 5]
```diff
@@ -3,6 +3,100 @@
...
+- Add `logging.metrics.namespaces` config option to control what metric groups are reported in logs. {pull}25727[25727]
+- Add sha256 digests to RPM packages. {issue}23670[23670]
+- Add new 'offline' docker image for Elastic Agent. {pull}27052[27052]
+- Add cgroups V2 support {pull}27242[27242]
+- update ECS field definitions to ECS 1.11.0. {pull}27107[27107]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:74]:
```suggestion
- Update ECS field definitions to ECS 1.11.0. {pull}27107[27107]
```

File: CHANGELOG.asciidoc [hunk 6]
```diff
@@ -3,6 +3,100 @@
...
+- The disk queue is now GA. {pull}27515[27515]
+- Add daemonset.name in pods controlled by DaemonSets {pull}26808[26808], {issue}25816[25816]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:76]:
```suggestion
- Add `daemonset.name` in pods controlled by DaemonSets {pull}26808[26808], {issue}25816[25816]
```

Comment (EamonnTP) [inline CHANGELOG.asciidoc:76]:
```suggestion
- Add `daemonset.name` in pods controlled by DaemonSets {pull}26808[26808], {issue}25816[25816]
```

File: CHANGELOG.asciidoc [hunk 7]
```diff
@@ -3,6 +3,100 @@
...
+
+*Filebeat*
+
+- Add new template functions and `value_type` parameter to `httpjson` transforms. {pull}26847[26847]
+- Add support to merge registry updates in the filestream input across multiple ACKed batches in case of backpressure in the registry or disk. {pull}25976[25976]
+- Add support to `decode_cef` for MAC addresses that do not contain separator characters. {issue}27050[27050] {pull}27109[27109]
+- Add new `hmac` template function for httpjson input {pull}27168[27168]
+- Update `tags` and `threatintel.indicator.provider` fields in `threatintel.anomali` ingest pipeline {issue}24746[24746] {pull}27141[27141]
+- Move AWS module and filesets to GA. {pull}27428[27428]
+- update ecs.version to ECS 1.11.0. {pull}27107[27107]
```
Comment (EamonnTP) [inline CHANGELOG.asciidoc:86]:
```suggestion
- Update ecs.version to ECS 1.11.0. {pull}27107[27107]
```

**Review by EamonnTP — APPROVED** [review]
@andresrc  Just minor suggestions.

**Pull request merged**